### PR TITLE
feat: Add protection role filter strings

### PR DESCRIPTION
**Pull request by FnrDev**

### CODE REVIEW

File: en/website.json [hunk 1]
```diff
@@ -660,5 +660,7 @@
     "billing_period": "Billing Period",
     "uninvited_subscription": "Uninvited Subscription",
     "unknown": "Unknown",
-    "upgrade": "Upgrade"
+    "upgrade": "Upgrade",
+    "protection_role_search": "SEARCH BY ROLE ID OR NAME",
```
Comment (Dramex) [inline en/website.json:664]: no need to specify "protection" in the key if the string can be used somewhere else

File: en/website.json [hunk 2]
```diff
@@ -660,5 +660,7 @@
...
+    "protection_type_here": "Type here .."
```
Comment (Dramex) [inline en/website.json:665]: no need to specify "protection" in the key if the string can be used somewhere else

**Review by Dramex — CHANGES_REQUESTED** [review]
remove protection from key to make it global since it can be used somewhere else

File: en/website.json (hunk 1 above)
Comment (FnrDev) [reply en/website.json]: fix (2d27b80a5a2471650826c347411b25ab2edf3f71)

File: en/website.json (hunk 2 above)
Comment (FnrDev) [reply en/website.json]: fixed (2d27b80a5a2471650826c347411b25ab2edf3f71)

**Comment by Dramex** [comment]
LGTM

**Pull request merged**